src/app/content/highlights/components/CardWrapper.tsx

```diff
@@ -37,6 +37,8 @@ const Wrapper = ({highlights, className, container, highlighter}: WrapperProps)
     () => highlights.find((highlight) => highlight.id === focusedId),
     [focusedId, highlights]);
   const prevFocusedHighlightId = React.useRef(focusedId);
+  // tslint:disable-next-line: no-empty
+  const setNewCardsPositionsRef = React.useRef(() => {});
```

Comment:
will it require tslint disable if you return `null` or `undefined`?

Reply:
it causes the ts error: `Type '() => void' is not assignable to type '() => undefined'.`

Reply:
You can type the ref to `() => void | undefined` or something